src/routes/v1/secret_test.go

```diff
@@ -55,19 +55,19 @@ func TestGetSpecificSecret(t *testing.T) {
 	assert.Equal(t, "test-secret", response.SecretName)
 }
 
-func TestPatchSecret(t *testing.T) {
-	patchRequest := types.PatchSecretRequest{
+func TestUpdateSecret(t *testing.T) {
+	updateRequest := types.UpdateSecretRequest{
 		Data: []types.KeyValue{{Key: "key2", Value: "ZmFrZQ=="}},
 	}
-	body, _ := json.Marshal(patchRequest)
-	request, _ := http.NewRequest("PATCH", "/v1/namespaces/default/secrets/test-secret", bytes.NewBuffer(body))
+	body, _ := json.Marshal(updateRequest)
+	request, _ := http.NewRequest("PUT", "/v1/namespaces/default/secrets/test-secret", bytes.NewBuffer(body))
```

Comment:
Same here, use `http.MethodPut`.

Reply:
Let me take care of it in a new PR - because it requires also changing other tests (in other files). I'll open a new PR and change all relevant files.